cmd/collectd-vsphere/main.go

```diff
@@ -49,42 +49,57 @@ func main() {
 			&cli.StringFlag{
 				Name:    "collectd-hostport",
 				Usage:   "the host:port for collectd",
-				EnvVars: []string{"COLLECTD_HOSTPORT"},
+				EnvVars: []string{"COLLECTD_VSPHERE_COLLECTD_HOSTPORT", "COLLECTD_HOSTPORT"},
 			},
 			&cli.StringFlag{
 				Name:    "collectd-username",
 				Usage:   "the username for collectd",
-				EnvVars: []string{"COLLECTD_USERNAME"},
+				EnvVars: []string{"COLLECTD_VSPHERE_COLLECTD_USERNAME", "COLLECTD_USERNAME"},
 			},
 			&cli.StringFlag{
 				Name:    "collectd-password",
 				Usage:   "the password for collectd",
-				EnvVars: []string{"COLLECTD_PASSWORD"},
+				EnvVars: []string{"COLLECTD_VSPHERE_COLLECTD_PASSWORD", "COLLECTD_PASSWORD"},
 			},
 			&cli.StringFlag{
 				Name:    "vsphere-url",
 				Usage:   "the URL for the vSphere API",
-				EnvVars: []string{"VSPHERE_URL"},
+				EnvVars: []string{"COLLECTD_VSPHERE_VSPHERE_URL", "VSPHERE_URL"},
 			},
 			&cli.BoolFlag{
 				Name:    "vsphere-insecure",
 				Usage:   "connect to vSphere without verifying TLS certs",
-				EnvVars: []string{"VSPHERE_INSECURE"},
+				EnvVars: []string{"COLLECTD_VSPHERE_VSPHERE_INSECURE", "VSPHERE_INSECURE"},
 			},
 			&cli.StringFlag{
 				Name:    "vsphere-cluster",
 				Usage:   "path to the vSphere cluster to monitor events on",
-				EnvVars: []string{"VSPHERE_CLUSTER"},
+				EnvVars: []string{"COLLECTD_VSPHERE_VSPHERE_CLUSTER", "VSPHERE_CLUSTER"},
+			},
+			&cli.StringSliceFlag{
+				Name:    "vsphere-clusters",
+				Usage:   "paths to the vSphere clusters to monitor events on",
```

Comment:
Good point, it's comma-separated, but it doesn't look like that shows in the help text at all:

```
   --vsphere-clusters value          paths to the vSphere clusters to monitor events on [$COLLECTD_VSPHERE_VSPHERE_CLUSTERS, $VSPHERE_CLUSTERS]
```

How about changing to

``` Go
Usage: "comma-separated paths to the vSphere…"
```

(and same for the other slice flags)?